MAUI/Barcode/two-dimensional-symbology.md

```diff
@@ -0,0 +1,273 @@
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode ErrorCorrectionLevel="High"/>          
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode() 
+    { 
+        ErrorCorrectionLevel = ErrorCorrectionLevel.High 
+    };
+
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code error correction level high](images/two-dimensional-symbology/maui-qr-code-error-correction-level-high.png)
+
+The data can be read from the damaged image based on the error correction level,
+
+![.NET MAUI Barcode Generator QR Code damaged](images/two-dimensional-symbology/maui-qr-code-damaged.png)
+
+### Input mode
+
+The `InputMode` property allows you to select a specific set of input characters. You may select the most suitable input mode. By default, its value is set as `Auto`.
+
+Numeric - supports from 0 to 9
+AlphaNumeric - supports 0 to 9, A to Z, space, $, %, *, +, -,., /, :
+Binary - supports Shift JIS characters.
+Auto - it calculates input mode automatically based on `CodeVersion` and `ErrorCorrectionLevel`.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+   <barcode:SfBarcodeGenerator Value="1263438892737643894930872"
+                               ShowText="True"
+                               HeightRequest="300"
+                               WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode InputMode="Numeric"/>
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "1263438892737643894930872";
+    barcode.Symbology = new QRCode() 
+    { 
+        InputMode = QRInputMode.Numeric 
+    };
+
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Input Mode As Numeric](images/two-dimensional-symbology/maui-qr-code-input-mode-numeric.png)
+
+### QR code version
+
+The `CodeVersion` property allows you to set various types of version for QR code from QRVersion enumeration. By default, its value is set as auto.
+
+The QR Code uses version from 1 to 40. Version 1 measures 21 modules x 21 modules, Version 2 measures 25 modules x 25 modules, and so on.
+
+The number of modules increases in steps of 4 modules per side up to Version 40 that measures 177 modules x 177 modules.
```

Comment:
The number of modules increases in steps of 4 modules per side up to Version 40, which measures 177 modules x 177 modules.

Reply:
Changes updated.